app/assets/stylesheets/report.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Report controller here.
```

Comment:
If its ok can we not have random empty scss files checked in? I know scaffolding generates them, but they're just IDE noise